src/localPipResolver.js

```diff
@@ -3,6 +3,8 @@
 var logger = require('pelias-logger').get('wof-admin-lookup');
 var createPIPService = require('pelias-wof-pip-service').create;
 
+let datapath;
```

Comment:
what is the advantage of using module scope here over dependency injection via function args?